README.md

```diff
@@ -354,19 +391,58 @@ tasks.koverVerify {
 ```
 </details>
 
+To add rules for code coverage checks for one specific module, you need to add a configuration to this module:
+
+<details open>
+<summary>Kotlin</summary>
+
+```kotlin
+tasks.koverModuleVerify {
+    rule {
+        name = "Minimal line coverage rate in percent"
+        bound {
+            minValue = 75
+       }
+    }
+}
+```
+</details>
+
+<details>
+<summary>Groovy</summary>
+
+```groovy
+tasks.koverModuleVerify {
+    rule {
+        name = "Minimal line coverage rate in percent"
+        bound {
+            minValue = 75
+        }
+    }
+}
+```
+</details>
+
+
 ## Tasks
-The plugin, when applied, automatically creates tasks for the module (all modules, if the project is multi-module, and it applied in root build script):
-- `koverXmlReport` - Generates code coverage XML report for all module's test tasks.
-- `koverHtmlReport` - Generates code coverage HTML report for all module's test tasks.
-- `koverReport` - Executes both `koverXmlReport` and `koverHtmlReport` tasks.
-- `koverCollectReports` - Collects reports from all submodules in one directory. Default directory is `$buildDir/reports/kover/all`, names for XML reports and dirs for HTML are projects names. Executing this task does not run `koverXmlReport` or `koverHtmlReport`, it only copies previously created reports if they exist to the output directory.
-- `koverVerify` - Verifies code coverage metrics based on specified rules. Always executes before `check` task.
+The plugin, when applied, automatically creates tasks for the module in which it is applied (usually this is the root module):
+- `koverHtmlReport` - Generates code coverage HTML report for all enabled test tasks in all modules.
+- `koverXmlReport` - Generates code coverage XML report for all enabled test tasks in all modules. 
+- `koverReport` - Executes both `koverXmlReport` and `koverHtmlReport` tasks.  Executes before `check` task if `generateReportOnCheck` is `true`.
```

Comment:
It's not specified what `generateReportOnCheck` is. Is it a gradle property? Or Kover extension property? Probably we shouldn't mention it at all.